isis/tests/BundleSettingsTests.cpp

```diff
@@ -0,0 +1,17 @@
+#include "BundleSettings.h"
+
+#include <gtest/gtest.h>
+
+class BoolTest : public ::testing::TestWithParam<bool> {
+  // Intentionally empty
+};
+
+TEST_P(BoolTest, validateNetwork) {
+  BundleSettings testSettings;
+  testSettings.setValidateNetwork(GetParam());
+  EXPECT_EQ(GetParam(), testSettings.validateNetwork());
+}
+
+INSTANTIATE_TEST_CASE_P(BundleSettings,
```

Comment:
The first parameter for this macro determines the beginning of the test name. So, this creates the following two tests

`BundleSettings/BoolTest.validateNetwork/false`
`BundleSettings/BoolTest.validateNetwork/true`

I'd recommend we use the class we are testing for this parameter so that the parameterized tests follow our test naming conventions.